promscale/distributed-tracing.md

```diff
@@ -0,0 +1,288 @@
+# Tracing
+Promscale supports tracing using the [OpenTelemetry][opentelemetry] protocol
+(OTLP). Promscale supports the entire OpenTelemetry trace data model including
+spans, events, and links.
+
+You can also use Jaeger and Zipkin traces with the OpenTelemetry Collector. The
+OpenTelemetry Collector ingests data from Jaeger and Zipkin instrumentation,
+converts it to OpenTelemetry, and sends it to Promscale using OTLP.
+
+You can visualize traces in Promscale using Grafana and the Jaeger UI.
+
+When you have tracing set up in Promscale, you can query the traces using SQL
+queries. This allows you to get deep insights from your tracing data, which can
+help you understand problems and identify optimizations for your applications.
+
+<highlight type="warning">
+Promscale tracing is currently in beta, and could have bugs. This feature might
+not be backwards compatible, and could be removed in a future release. Use this
+feature at your own risk, and do not use any experimental features in
+production.
+</highlight>
+
+To enable tracing in Promscale 0.7.0 and later, use the
+`-enable-feature=tracing` flag. For more information about installing tracing
+support with tobs, see [the tobs installation section][tobs-tracing]. For more
+information about installing tracing support with Docker, see
+[the Docker installation section][docker-tracing].
+
+Please help us improve Promscale by
+[sharing your feedback](https://github.com/timescale/promscale/discussions/916)
+and [reporting issues](https://github.com/timescale/promscale/issues/new).
+
+## Ingest traces
+Promscale has native support for ingesting OpenTelemetry traces. You can send
+OpenTelemetry traces directly to Promscale using the
+[OpenTelemetry][opentelemetry] protocol (OTLP). For the best workflow, use OTLP for
+tracing in Promscale, because it centralizes management of all your telemetry
+data. This helps you define processing rules and manage where to send the
+data.
+
+Jaeger and Zipkin trace formats are supported using the OpenTelemetry Collector
```

Comment:
What's your reasoning for this comma?

Reply:
Oxford comma again because Jaeger, Zipkin, Datadog , and others are different tracing tools ....

Reply:
I can't see how it could be confused as a combined format, or how the comma prevents that. How about doing something like this instread:

```suggestion
Both Jaeger and Zipkin trace formats are supported using the OpenTelemetry Collector
```

Reply:
Thank you !!! you made it so clear....